Makefile

```diff
@@ -0,0 +1,21 @@
+NPM = /usr/bin/npm
+
+.PHONY: all build archive start
+
+all: build
+
+build: $(NPM) node_modules
+	$(RM) -r assert/
```

Comment:
I guess this is a typo: assert -> assets.

Reply:
Yup, fixed - I've written lots of asserts in Rust recently :).